docs/DeveloperGuide.md

```diff
@@ -257,11 +257,26 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
+* Name: Kevin Bernard Long Zheng Wei
+* Age: 28
+* Gender: Male
+* Marital status: Single
+* Country: Singapore
+* Habits: Sleeping, Netflix, Gym, Gaming
+* Lifestyle: Code, eat, sleep, give part time tuition
```

Comment:
Shouldn't the target user be a full-time tutor?

Reply:
Yes it should, forgot to change it